plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -985,8 +985,8 @@ function test(packageFolder) {
   return runScriptOnPackages(packageFolder, 'test', script, { UNDER_TEST: 'true' });
 }
 
-const headlampPluginBin = fs.realpathSync(process.argv[1]);
-console.log('headlampPluginBin path:', headlampPluginBin);
+// const headlampPluginBin = fs.realpathSync(process.argv[1]);
```

Comment:
@illume This is failing the tests where the headlamp-plugin command is expected to return a json response. Is it okay to remove this or does it server any crucial purpose?

Reply:
That's fine to comment out. It is just for debugging.